rpc/src/rpc_pubsub_service.rs

```diff
@@ -43,7 +44,7 @@ pub struct PubSubConfig {
     pub queue_capacity_items: usize,
     pub queue_capacity_bytes: usize,
     pub worker_threads: usize,
-    pub notification_threads: Option<usize>,
+    pub notification_threads: Option<NonZeroUsize>,
```

Comment:
I initially changed this to just a `usize`; there is a default value at the CLI level so we no longer needed to pass the `Option<_>` down.

However, I think `Option<NonZeroUsize>` conveys a bit more meaning; we are either disabling the service (`None`) or we're enabling the service with `N` threads (`Some(N)`) where `N > 0`